bigquery_storage/google/cloud/bigquery_storage_v1beta1/gapic/big_query_storage_client.py

```diff
@@ -288,7 +290,7 @@ def create_read_session(
         except AttributeError:
             pass
         else:
-            routing_metadata = google.api_core.gapic_v1.routing_header.to_grpc_metadata(  # pragma: no cover
```

Comment:
These get taken away, but it looks like coverage is still OK. @googleapis/api-bigquery Is it alright to let synth remove this?

Reply:
We really should get the codegen to drop the whole `try: ... except: ...` when constructing the header with simple types:  no AttributeError is possible.

Reply:
LGTM. I think the coverage config has changed since those were first added.